admiral/pkg/clusters/serviceentry.go

```diff
@@ -189,19 +190,98 @@ func createServiceEntryForNewServiceOrPod(env string, sourceIdentity string, rem
 			//} else {
 			//	log.Errorf(LogErrFormat, "Create", istioModel.VirtualService.Type, virtualServiceName, rc.ClusterID, err)
 			//}
+			if common.GetWorkloadSidecarUpdates() == "enabled" {
+				for _, val := range dependents.Map() {
```

Comment:
I think this `for` loop should be outside the if block since the DependencyNamespaceCache can be used for other purposes.